setup.py

```diff
@@ -14,21 +14,22 @@
     "accept-rom-license": ["autorom[accept-rom-license]~=0.4.2"],
     "box2d": ["box2d-py==2.3.5", "pygame==2.1.0"],
     "classic_control": ["pygame==2.1.0"],
-    "mujoco": ["mujoco_py>=1.50, <2.0"],
+    "mujoco_py": ["mujoco_py<2.2,>=2.1"],
+    "mujoco": ["mujoco>=2.1.3", "imageio>=2.14.1"],
```

Comment:
Would you mind making this mujoco~=2.1.3, so that version 2.2 isn't automatically used when it's released?

We probably won't make backwards-incompatible changes, but we wouldn't want to accidentally break all Gym users.

Reply:
Actually on second thought, maybe it's better for you to completely pin the version, since you're going to be using mujoco as a benchmark.

We're going to make small physics improvements between versions, and you'll want to decide whether to pick them up.

For example, in 2.1.4 (released yesterday), we fixed a bug in quaternion maths which happens for rotation angles smaller than 1e-8. It's making the physics more correct, but maybe you want to check agent performance before you move to the new version.